lib/galaxy/tools/errors.py

```diff
@@ -208,7 +208,7 @@ def create_report( self, user, email='', message='', **kwd ):
 
         # Escape all of the content  for use in the HTML report
         for parameter in report_variables.keys():
-            report_variables[parameter] = cgi.escape(report_variables[parameter])
+            report_variables[parameter] = cgi.escape(str(report_variables[parameter]))
```

Comment:
Not sure if the following can happen, but `str()` may fail if `report_variables[parameter]` is of type unicode:
```
>>> foo = u'é'
>>> str(foo)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe9' in position 0: ordinal not in range(128)
```
So I'd suggest either:
```
import six
...
            report_variables[parameter] = cgi.escape(report_variables[parameter] if isinstance(report_variables[parameter], six.string_types) else str(report_variables[parameter]))
```
or:
```
from galaxy.util import unicodify
...
            report_variables[parameter] = cgi.escape(unicodify(report_variables[parameter]))
```